src/js/components/RangeInput/StyledRangeInput.js

```diff
@@ -246,6 +246,16 @@ const StyledRangeInput = styled.input`
             )};
         }
       `}
+    ${(props) =>
+      props.focus &&
+      !props.focusIndicator &&
+      css`
+        box-shadow: 0px 0px 0px 2px
```

Comment:
Would it be possible to extract this as a `hoverStyle()` function so it's not copy and pasted from the lines above?